### ISSUE TITLE
[nbd] XFS (nbd0): metadata I/O error

### ISSUE THREAD

**Issue by lnsyyj** [vitalif/vitastor#14]
When I use nbd to mount the vitastor image and read and write, the following error occurs

```
[Mon Jul 19 10:37:34 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:34 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:34 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:34 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:34 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:34 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:34 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:34 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:34 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:34 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:34 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:34 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:34 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:34 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:34 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:34 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:34 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:34 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:34 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:34 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:34 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:34 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:34 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:34 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:34 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:34 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:34 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:34 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:34 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:34 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:34 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:34 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:34 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:34 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:34 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:34 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:34 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:34 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:34 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:34 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:34 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:34 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:34 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:34 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:35 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:35 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:35 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:35 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:35 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:35 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:35 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:35 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:35 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:35 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:35 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:35 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:35 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:35 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:35 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:35 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:35 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:35 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:35 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:35 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:35 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:35 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:35 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:35 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:35 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:35 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:35 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:35 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:35 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:35 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:35 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:35 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:35 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:35 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:35 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:35 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:35 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:35 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:35 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:35 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:35 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:35 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:35 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:35 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:35 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:35 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:35 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:35 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:35 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:35 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:35 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:35 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:35 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:35 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:35 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:35 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:35 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:35 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:35 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:35 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:35 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:35 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:35 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:35 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Metadata corruption detected at xfs_dinode_verify.part.0+0x172/0x640 [xfs], inode 0x14506b6d dinode
[Mon Jul 19 10:37:35 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:37:35 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:37:35 2021] 00000000: 49 4e 81 a4 03 02 00 00 00 00 00 00 00 00 00 00  IN..............
[Mon Jul 19 10:37:35 2021] 00000010: 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:37:35 2021] 00000020: 60 f4 e3 3f 07 da c4 4e 60 f4 e3 3f 07 da c4 4e  `..?...N`..?...N
[Mon Jul 19 10:37:35 2021] 00000030: 60 f4 e3 3f 07 da c4 4e 00 00 00 00 00 00 1b f4  `..?...N........
[Mon Jul 19 10:37:35 2021] 00000040: 00 00 00 00 00 00 00 02 00 00 00 00 00 00 00 01  ................
[Mon Jul 19 10:37:35 2021] 00000050: 00 00 00 02 00 00 00 00 00 00 00 00 ca c6 e6 ca  ................
[Mon Jul 19 10:37:35 2021] 00000060: ff ff ff ff dd d6 04 ae 00 00 00 00 00 00 00 05  ................
[Mon Jul 19 10:37:35 2021] 00000070: 00 00 00 02 00 01 da a8 00 00 00 00 00 00 00 00  ................
[Mon Jul 19 10:38:42 2021] XFS (nbd0): Corruption warning: Metadata has LSN (7:27) ahead of current LSN (3:28176). Please unmount and run xfs_repair (>= v4.3) to resolve.
[Mon Jul 19 10:38:42 2021] XFS (nbd0): Metadata CRC error detected at xfs_inobt_read_verify+0x12/0x90 [xfs], xfs_inobt block 0x12e423e8 
[Mon Jul 19 10:38:42 2021] XFS (nbd0): Unmount and run xfs_repair
[Mon Jul 19 10:38:42 2021] XFS (nbd0): First 128 bytes of corrupted metadata buffer:
[Mon Jul 19 10:38:42 2021] 00000000: 54 5a 69 66 32 00 00 00 00 00 00 00 00 00 00 00  TZif2...........
[Mon Jul 19 10:38:42 2021] 00000010: 00 00 00 00 00 00 00 07 00 00 00 07 00 00 00 1b  ................
[Mon Jul 19 10:38:42 2021] 00000020: 00 00 00 3b 00 00 00 07 00 00 00 1a 80 00 00 00  ...;............
[Mon Jul 19 10:38:42 2021] 00000030: 91 05 fc 00 da 62 04 38 4c 9f 27 c8 4d 97 2b f8  .....b.8L.'.M.+.
[Mon Jul 19 10:38:42 2021] 00000040: 4e 7d e2 78 4e fd 8b b8 4f 77 0d f8 50 66 fe f9  N}.xN...Ow..Pf..
[Mon Jul 19 10:38:42 2021] 00000050: 51 60 2a 79 52 46 e0 f9 53 40 0c 79 54 26 c2 f9  Q`*yRF..S@.yT&..
[Mon Jul 19 10:38:42 2021] 00000060: 55 1f ee 79 56 06 a4 fa 56 ff d0 7a 57 e6 86 fa  U..yV...V..zW...
[Mon Jul 19 10:38:42 2021] 00000070: 58 df b2 7b 59 c6 68 fb 5a bf 94 7b 5b af 85 7b  X..{Y.h.Z..{[..{
[Mon Jul 19 10:38:42 2021] XFS: metadata IO error: 1716 callbacks suppressed
[Mon Jul 19 10:38:42 2021] XFS (nbd0): metadata I/O error in "xfs_btree_read_buf_block.constprop.0+0x95/0xd0 [xfs]" at daddr 0x12e423e8 len 8 error 74
[Mon Jul 19 10:38:42 2021] XFS (nbd0): xfs_do_force_shutdown(0x1) called from line 296 of file fs/xfs/xfs_trans_buf.c. Return address = 00000000f86604ca
[Mon Jul 19 10:38:42 2021] XFS (nbd0): I/O Error Detected. Shutting down filesystem
[Mon Jul 19 10:38:42 2021] XFS (nbd0): Please unmount the filesystem and rectify the problem(s)
[Mon Jul 19 10:38:42 2021] XFS (nbd0): xfs_difree_inobt: xfs_inobt_lookup() returned error -117.

```

**Comment by vitalif** [comment]
How can I reproduce it?

**Comment by lnsyyj** [comment]
My steps are:
```
1. Create a pool
2. Create inode 200G
3. Via vitastor-nbd map inode
4. Create a file system mkfs.xfs /dev/nbd0
5. Mount the file system mount /dev/nbd0 /root/nbd0/
6. Then I copied a lot of code to the /root/nbd0/ directory
cp -r github/ /root/nbd0/github, cp -r github/ /root/nbd0/github2, cp -r github/ /root/nbd0/github3...
```
So it will reappear

**Comment by vitalif** [comment]
Hi, I recently fixed a very simple "splitbrain" issue (commit aa436027c8aa1a788a37f12e118fe46865cb70f0) which I missed in previous versions. It could easily manifest as data corruption if you had 2 osds and if you were stopping them one after another to test failover :-). May it be similar to your case?

**Comment by lnsyyj** [comment]
Hi @vitalif ,
The problem is solved in version 0.6.10, I think it can be closed.

**Comment by vitalif** [comment]
Cool
